index.html

```diff
@@ -935,8 +951,11 @@ <h2>
           <li>
             <p>
               Otherwise, present a user interface to allow the user to interact
-              with the <var>handlers</var>. The user agent SHOULD prioritize
-              the preference of the user when presenting payment methods.
+              with the <var>handlers</var>. Optionally, if
+              <var>request</var>.<a>[[\uiLang]]</a> is not null, localize the
+              user interface to match, as closely as possible, the language of
+              <a>[[\uiLang]]</a>. The user agent SHOULD prioritize the
```

Comment:
Should we state more about what to do if [[uiLang]] is null? For example using the [language](https://html.spec.whatwg.org/#language) of [the body element](https://html.spec.whatwg.org/#the-body-element-2)?